### PR TITLE
update to beta version

### PR DESCRIPTION
**Pull request by justin808**
This change is [Reviewable](https://reviewable.io/reviews/shakacode/react-webpack-rails-tutorial/474)

### CODE REVIEW

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/* eslint no-console:0 */
```
Comment (justin808) [inline app/javascript/packs/application.js:1]: all files in this directory should get removed...stick to your old custom one...

you ***did not need to run the webpack installer***

it was already there...

File: babel.config.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+module.exports = function(api) {
```
Comment (justin808) [inline babel.config.js:1]: more files from the installer that should get removed...

File: client/webpack.server.rails.build.config.js [hunk 3]
```diff
@@ -1,90 +1,38 @@
...
+const devBuild = process.env.NODE_ENV === 'production' ? 'production' : 'development';
 const webpack = require('webpack');
-const webpackConfigLoader = require('react-on-rails/webpackConfigLoader');
-
-const configPath = resolve('..', 'config');
-const { output } = webpackConfigLoader(configPath);
 
-const devBuild = process.env.NODE_ENV !== 'production';
+// React Server Side Rendering webpacker config
+// Builds a Node compatible file that React on Rails can load, never served to the client.
```
Comment (justin808) [inline client/webpack.server.rails.build.config.js:7]: why is this file changing?

for just updating the gem, don't change these

do the upgrade steps suggest you need this?

File: config/webpack/development.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+process.env.NODE_ENV = process.env.NODE_ENV || 'development'
```
Comment (justin808) [inline config/webpack/development.js:1]: all these need to be removed b/c you have a custom webpack config

future PR will put these in

File: package.json [hunk 5]
```diff
@@ -39,8 +39,14 @@
     "url": "https://github.com/shakacode/react-webpack-rails-tutorial/issues"
   },
   "homepage": "https://github.com/shakacode/react-webpack-rails-tutorial",
-  "dependencies": {},
-  "devDependencies": {},
+  "dependencies": {
```
Comment (justin808) [inline package.json:42]: this makes no sense

File: postcss.config.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
```
Comment (justin808) [inline postcss.config.js:1]: another file you don't need right now

File: package.json (hunk 5 above)
Comment (hussainakram) [reply package.json:42]: I followed latest webpack installation steps: https://github.com/rails/webpacker#installation

**Pull request closed without merging**